Template/Tests/{PROJECT}Tests/{PROJECT}Tests.swift

```diff
@@ -27,5 +27,22 @@ import XCTest
 import {PROJECT}
 
 class {PROJECT}Tests: XCTestCase {
-    
+    func testExample() {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+        //// XCTAssertEqual({PROJECT}().text, "Hello, World!")
+    }
+
+
+}
+
+#if os(Linux)
+extension {PROJECT}Tests {
+    static var allTests : [(String, ({PROJECT}Tests) -> () throws -> Void)] {
+        // FIXME: Should return a list of test case (name, function) tuples
```

Comment:
On the `//FIXME` in the templated code, I guess it's an interesting point on whether to leave it in... since this list is separately "hand" maintained for Linux testing? 

I deleted it in this PR though.